apps/experiments/views/experiment.py

```diff
@@ -239,15 +245,19 @@ def clean(self):
         bot_type = cleaned_data["type"]
         if bot_type == "llm":
             cleaned_data["assistant"] = None
+            cleaned_data["pipeline"] = None
             if not cleaned_data.get("prompt_text"):
                 errors["prompt_text"] = "Prompt text is required unless you select an OpenAI Assistant"
             if not cleaned_data.get("llm_provider"):
                 errors["llm_provider"] = "LLM Provider is required unless you select an OpenAI Assistant"
             if not cleaned_data.get("llm"):
                 errors["llm"] = "LLM is required unless you select an OpenAI Assistant"
-        else:
+        elif bot_type == "assistant":
             if not cleaned_data.get("assistant"):
                 errors["assistant"] = "Assistant is required when creating an assistant experiment"
+        elif bot_type == "pipeline":
```

Comment:
Should this set `assistant` to None in `cleaned_data`? (also above set `pipeline` to None if it's an assistant)?